contributors/devel/architectural-roadmap.md

```diff
@@ -613,7 +613,7 @@ Automation APIs and functions:
 * Dynamic load-balancer provisioning
 
 * NIY: The
-  [PodPreset](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/pod-preset.md)
+  [PodPreset](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/service-catalog/pod-preset.md)
```

Comment:
same repo. This can be:
`[PodPreset](/contributors/design-proposals/service-catalog/pod-preset.md)`